next/components/common/Icons.tsx

```diff
@@ -86,12 +86,29 @@ export const StarIcon = () => (
 
 export const GitHub = () => (
     <svg
-        className="w-6 h-6 fill-base-content"
+        className="w-5 h-5 fill-slate-400"
```

Comment:
It looks like you replaced `base-content` with `slate-400`. To be clear, `base-content` refers to a color in the daisyui color theme that changes based on the data theme (dark or light mode). What was the reason for changing this back to slate-400? Maybe once I understand that, I can help figure out any color issues.

Reply:
Oh, it was an uninformed decision. I have changed it back to base-content with room for customization. 

Is this okay? 

### Icons.tsx
```
export const LinkedIn = ({color}) => (
    <svg 
        className={`w-5 h-5 fill-${color ? color : 'base-content'}`}
        viewBox="0 0 48 48"
    >
    // redacted for brevity
    </svg>
);
```
### page.tsx
```
let iconColor = '';
<LinkedIn color={iconColor} />
```

Reply:
I love that you're generalizing the icon's color! This is a great start, and it got me thinking of a few additional considerations. 
- What datatype should the component expect the "color" value to be passed in as? To US, it's obvious it's a string, but we should make it more clear. Typescript in the props can help with this.
- Okay, so now we have types and it's clear it needs to be a string... What strings will be successfully parsed by the component? 

As it stands, only strings for colors already defined by either tailwind or the daisyUI config will work. That is, "primary", "base-100", "emerald-400". It's fine to retain this limitation (because it means developers need to go out of their way to use a random color that might not match the palette), but documentation of this limitation/constraint would be beneficial to future developers and people using the component. 

You could use JSdocs for this: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Reply:
Another option is to just have the whole className string passed in as a prop, which I think is the optimal solution as it lets us use the same icon svg with a lot more flexibility across the app.